app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -53,7 +53,7 @@ def cluster(cluster)
     end
 
     def clusters
-      dc.clusters.map(&:full_path).sort
+      dc.clusters.map(&:full_path).sort rescue []
```

Comment:
Can we log a warning in the Rails log with that? Just to be sure we don't swallow the exception. Put something like "VmWare returned zero DC count" or similar there. :+1: